normandy/classifier/api.py

```diff
@@ -6,7 +6,8 @@
 
 
 class FetchBundle(views.APIView):
-
+    authentication_classes = []
+    permission_classes = []
```

Comment:
I think it's a good thing to do even for the API, but I can change it anyway if you feel strongly about it.